app/models/census_employee.rb

```diff
@@ -27,6 +27,7 @@ class CensusEmployee < CensusMember
   field :employment_terminated_on, type: Date
   field :coverage_terminated_on, type: Date
   field :aasm_state, type: String
+  field :expected_selection, type: String, default: "Enroll"
```

Comment:
You should add the following for this field:

1. a set of enumerated constants that represent the allowed value
2. a validation for the column value
3. a spec which checks rejection of invalid values